src/v2/components/Transactions/index.jsx

```diff
@@ -45,7 +45,7 @@ const TransactionsPage = ({match}: {match: Match}) => {
 
   return (
     <Container>
-      <SectionHeader title="Transactions">
+      <SectionHeader title="Recent Transactions">
```

Comment:
(added "recent" qualifier to disambiguate transaction count here vs. all time transactions)